modules/swagger-parser/src/test/java/io/swagger/parser/SwaggerResolverTest.java

```diff
@@ -40,7 +41,7 @@ public void testSwaggerResolver(@Injectable final Swagger swagger,
             result = definitionsProcessor;
             times = 1;
 
-            new PathsProcessor(cache, swagger);
+            new PathsProcessor(cache, swagger, withInstanceOf(SwaggerResolver.Settings.class));
```

Comment:
I don't like this.  What else would be passed in the `PathsProcessor`?  Is this expected to be an extension point?